spec/connect/yast_spec.rb

```diff
@@ -4,6 +4,17 @@
 
   subject { SUSE::Connect::YaST }
 
+  describe '#defaults' do
+    it 'returns expected keys' do
+      expect(subject.defaults).to have_key(:config_file_path)
```

Comment:
This is good that we check for the key, but what we want to ensure is that defaults key value is the same as a Connect constant value see 4) in previous comment